docs/pages/Cardinal/API-Reference/World.mdx

```diff
@@ -0,0 +1,529 @@
+    }
+}
+```
+
+## NewWorld
+
+`NewWorld` creates a new `World` object.
+
+```go
+func NewWorld(s storage.WorldStorage, opts ...Option) (*World, error)
+```
+
+### Parameters
+| Parameter | Type                        | Description                                               |
+|-----------|-----------------------------|-----------------------------------------------------------|
+| s         | storage.WorldStorage       | The storage backend to be used for the world.             |
+| opts      | ...Option                  | Optional functional options for configuring the world.    |
+
+### Return Values
+
+| Type  | Description                                      |
+|--------------|--------------------------------------------------|
+| *World       | A pointer to the newly created World instance.  |
+| error        | An error indicating any issues during creation. |
+
+### Options
+
+#### WithAdapter
+
+The `WithAdapater` world option enables `World` objects to read transactions from the EVM base shard, and rebuild game state from those transactions.
+
+```go
+func WithAdapter(adapter shard.Adapter) Option
+```
+
+| Parameter | Type              | Description                              |
+|-----------|-------------------|------------------------------------------|
+| adapter   | shard.Adapter     | The adapter to be associated with World. |
+
+#### WithReceiptHistorySize
+
+The `WithReceiptHistorySize` option informs the `World` object on how many ticks the `World` should keep receipts for. For example, if we are on tick 40, and have a receipt history size of 5, the `World` will have receipts for ticks 35-39. On tick 41, the `World` will have receipts for ticks 36-40.
+
+```go
+func WithReceiptHistorySize(size int) Option
+```
+
+| Parameter | Type     | Description                                           |
+|-----------|----------|-------------------------------------------------------|
+| size      | int      | The size of the receipt history to be set for World. |
+
+
+#### WithNamespace
+
+The `WithNamespace` allows the `World` to have a custom Namespace. Namespaces are used to differentiate `World` objects that post data to the EVM base shard.
+
+```go
+func WithNamespace(id string) Option
+```
+| Parameter | Type    | Description                                   |
+|-----------|---------|-----------------------------------------------|
+| ns        | string  | The namespace to be set for World. |
+
+
+## Methods
+
+### AddSystems
+
+`AddSystems` adds one or more systems to the `World`. Systems are executed in the order of which they were added to the world.
+
+```go
+func (w *World) AddSystems(s ...System)
+```
+
+#### Example
+
+```go
+package main
+
+import (
+    "pkg.world.dev/world-engine/cardinal/ecs"
+
+    "github.com/my-username/my-world-engine-project/systems"
+)
+
+func main() {
+    // ... world setup ...
+
+    world.AddSystems(systems.MoveSystem, systems.HealthRegenSystem, systems.AttackSystem)
```

Comment:
Add a comment here that mention that MoveSystem will run first, followed HealthRegenSystem, etc ...

Reply:
[docs: system ordering note](https://github.com/Argus-Labs/world-engine/pull/200/commits/9d5e32ba8ee83d3d51ada018e5e543a5788dd2aa)